sqldb/sqlite.go

```diff
@@ -26,6 +26,16 @@ const (
 	sqliteTxLockImmediate = "_txlock=immediate"
 )
 
+var (
+	// sqliteSchemaReplacements is a map of schema strings that need to be
+	// replaced for sqlite. This is needed because sqlite doesn't directly
+	// support the BIGINT type for primary keys, so we need to replace it
+	// with INTEGER.
+	sqliteSchemaReplacements = map[string]string{
+		"BIGINT PRIMARY KEY": "INTEGER PRIMARY KEY",
+	}
+)
```

Comment:
Good question, I think we generally use a common SQL "dialect" for our schema definitions, but then we alter it slightly for both SQLite and Postgres. I believe we don't necessarily prefer one over the other, except for testing where a SQLite db is much faster to spin up. Perhaps we just reused code from Taproot Assets and these replaces come from there.

Reply:
So is it fair to say then that the default queries are written against the SQL standard then? If that's the case should we change this commentary on the `postgresSchemaReplacements`

```
        // replaced for postgres. This is needed because we write the schemas
	// to work with sqlite primarily, and postgres has some differences.
```

Reply:
Done, PTAL.